temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
Avoid using print statements for logging. Use the logging module instead for better control over logging levels and outputs.
```
Replace `print("hello")` with:
```python
import logging
logging.info("hello")
```
```